decentraland/unity-explorer
# feat: added remote player pins

## What does this PR change?

This PR adds the remote player markers in the map

...

## How to test the changes?



1. Launch the explorer
2. Open the map
3. Verify you see the red circle indicating players around the map even far from you
4. The far player position is updated only when toggling on and off the player layer or teleporting

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
